### DISCUSSION TITLE
Remove hosts/network in analyst via so-allow

### DISCUSSION THREAD

**Discussion by anthonyrod72** [Security-Onion-Solutions/securityonion#5657] [General] [answered]
how can I remove hosts/networks added in so-allow by mistake?

**Comment by clairmont32** [comment, marked as answer]
You can do this with `sudo so-firewall excludehost analyst 192.168.1.255`.  The docs dont explicitly cover this so it likely would be a good addition to the example section.

https://docs.securityonion.net/en/latest/firewall.html?highlight=so-firewall

**Reply by dougburks** [reply]
Added:
https://docs.securityonion.net/en/latest/firewall.html#removing-a-host-or-network
